pyspectral/near_infrared_reflectance.py

```diff
@@ -219,15 +219,9 @@ def reflectance_from_tbs(self, sun_zenith, tb_near_ir, tb_thermal, **kwargs):
         else:
             is_masked = False
 
-        if np.isscalar(tb_near_ir):
-            tb_nir = array([tb_near_ir, ])
-        else:
-            tb_nir = asanyarray(tb_near_ir)
-
-        if np.isscalar(tb_thermal):
-            tb_therm = array([tb_thermal, ])
-        else:
-            tb_therm = asanyarray(tb_thermal)
+        tb_nir = get_as_array(tb_near_ir)
+        tb_therm = get_as_array(tb_thermal)
+        sun_zenith = get_as_array(sun_zenith)
```

Comment:
Just curious, does `np.asanyarray` do the same thing for scalars? This custom function may not be needed.

Reply:
@djhoese Well it doesn't return the same thing. I think I could replace the call to `array` with `asanyarray`. But to have consistent return values I would keep the function:
```
In [1]: from numpy import where, logical_or, asanyarray, array
In [2]: x = asanyarray(2.3)
In [3]: y = array([2.3])
In [4]: x
Out[4]: array(2.3)
In [5]: y
Out[5]: array([2.3])
In [6]: x.shape, y.shape
Out[6]: ((), (1,))
```